chart/templates/secret.yaml

```diff
@@ -13,6 +13,9 @@ data:
   {{- if .Values.config.baaahThreadiness }}
   baaahThreadiness: {{ .Values.config.baaahThreadiness | b64enc }}
     {{- end }}
+  {{- if .Values.config.bootstrapToken }}
+  bootstrapToken: {{ .Values.config.bootstrapToken | b64enc }}
```

Comment:
`obotBootstrapToken` is already in here.. this config is either a dupe, or you meant to add config for disabling the bootstrap user here.

Reply:
Yep, that was an accident. Thanks. I just pushed the fix.